src/display.c

```diff
@@ -329,6 +331,9 @@ PyImaging_GrabScreenWin32(PyObject* self, PyObject* args)
     HDC screen, screen_copy;
     DWORD rop;
     PyObject* buffer;
+    HANDLE dpiAwareness;
+	HMODULE user32 = LoadLibraryA("User32.dll");
+	Func_SetThreadDpiAwarenessContext SetThreadDpiAwarenessContext_function;
```

Comment:
Can someone suggest a better name here? This the best I could do but it's ridiculously long.

Reply:
At least it's descriptive! :)